src/views/shared/external-link/external-link.view.tsx

```diff
@@ -0,0 +1,17 @@
+import { FC, PropsWithChildren } from "react";
+
+import { useExternalLinkStyles } from "src/views/shared/external-link/external-link.styles";
+
+interface ExternalLinkProps {
+  href: string;
+}
+
+export const ExternalLink: FC<PropsWithChildren<ExternalLinkProps>> = ({ children, href }) => {
+  const classes = useExternalLinkStyles();
+
+  return (
+    <a className={classes.link} href={href} rel="noopener noreferrer" target="_blank">
+      {children}
+    </a>
+  );
+};
```

Comment:
We can also consider replacing these now that we have a component:

<img width="660" alt="image" src="https://user-images.githubusercontent.com/15896177/227911957-d7856b44-3f8b-4e43-97bc-fdc342f19a5e.png">

Reply:
I've updated just one, the other ones have different styles (they are links inside a box) and will take more time to adjust them.